etc/config/cmakescript.amazon.properties

```diff
@@ -1,13 +1,37 @@
-compilers=cmake-3_26_1:cmake-3_27_9:cmake-3_28_0
+compilers=cmake-3_26_1:cmake-3_26_4:cmake-3_26:cmake-3_27_9:cmake-3_27:cmake-3_28_0:cmake-3_28_4:cmake-3_28:cmake-3_29_2:cmake-3_29:cmake-importstd
 isSemVer=true
 baseName=cmake
 defaultCompiler=cmake-3_28_0
 
 compiler.cmake-3_26_1.exe=/opt/compiler-explorer/cmake-v3.26.1/bin/cmake
 compiler.cmake-3_26_1.semver=3.26.1
 
+compiler.cmake-3_26_4.exe=/opt/compiler-explorer/cmake-v3.26.4/bin/cmake
+compiler.cmake-3_26_4.semver=3.26.4
+
+compiler.cmake-3_26.exe=/opt/compiler-explorer/cmake-v3.26/bin/cmake
+compiler.cmake-3_26.semver=3.26.4
```

Comment:
this is the same semver as above. It should be unique

Reply:
Ah. The idea was to give the option of using "the latest CMake 3.x" option. I'll remove them here.